open-defi-api.yaml

```diff
@@ -712,6 +712,73 @@ paths:
               schema:
                 $ref: "#/components/schemas/GenericErrorModel"
 
+  '/tokens':
+    x-pyramid-route-name: web_tokens
+    get:
+      operationId: web_tokens
+      summary: Information about supported tokens.
+      tags:
+        - Token
+      parameters:
+        - name: chain_slug
+          in: query
+          description: Blockchain slug.
+          required: false
+          schema:
+            type: string
+          example: "ethereum"
+        - name: exchange_slug
+          in: query
+          description: |
+             Exchange slug or exchange address. Providing this parameter requires
+             providing `chain_slug` as well.
+
+             Currently only Uniswap-like exchanges are supported.
+          required: false
+          schema:
+            type: string
+          examples:
+            "exchange slug":
+              value: "sushiswap"
+            "exchange address":
```

Comment:
Here are two examples, but only exchange_slug defined. Also `exchange slug` in the example spelled with the space.

Reply:
Both examples refer to the `exchange_slug` parameter, they just demonstrate two different kind of values that can be provided, i.e. either a slug or an address. The quoted names (with spaces) are jsut the human-readable values that appear in the "Examples" dropdown when rendered.

Or did I misunderstand the question?